lib/tokenizers.js

```diff
@@ -165,11 +186,24 @@ async function _getCurrentTokenizer() {
   return tokenizer;
 }
 
-async function _readCurrentTokenizerRecord() {
+export async function _readCurrentTokenizerRecord({explain = false} = {}) {
+  // the query of 'tokenizer.current' is properly indexed. No additional queries
+  // should be added here without ensuring that additional queries are properly
+  // indexed first.
```

Comment:
So last suggestion on these comments -- I'd say remove all of the comments that simply repeat good database code hygiene advice.